packages/xo-web/src/common/xo/index.js

```diff
@@ -300,6 +300,8 @@ export const subscribeCurrentUser = createSubscription(() => xo.refreshUser())
 
 export const subscribeAcls = createSubscription(() => _call('acl.get'))
 
+export const subscribeHvSupportedVersions = createSubscription(() => _call('xoa.getHVSupportedVersions'))
```

Comment:
This will fail if `xo-server-xoa` plugin isn't installed and loaded (e.g.: when XO is installed from the sources).